app/services/search/query_builders/chat_channel_membership.rb

```diff
@@ -40,10 +40,12 @@ def build_queries
       end
 
       def filter_conditions
+        terms_filter_keys = %i[viewable_by status]
```

Comment:
Should this also be constant?

Reply:
Probably yes. I couldn't come up with a proper name so I kept it there :D

Reply:
I'm gonna go wild here and suggest `TERMS_FILTER_KEYS` 😉